bool initialize
bool analyze_previous
float64 push_angle
---
tabletop_pushing/PushVector push
geometry_msgs/Vector3 moved
bool no_push
